### PR TITLE
changing used of signal method to unsupervised method

### PR DESCRIPTION
**Pull request by girishvn**
Test w/ PURE_UNSUPERVISED.YAML
(W/ preprocessing True). The output is as follows:

```===Signal Method ( ICA ) Predicting ===
  0%|                                                    | 0/59 [00:00<?, ?it/s]/mmfs1/gscratch/ubicomp/girishvn/rppg/rPPG-Toolbox/unsupervised_methods/methods/ICA_POH.py:44: ComplexWarning: Casting complex values to real discards the imaginary part
  BVP_F = signal.filtfilt(B, A, BVP_I.astype(np.double))
100%|███████████████████████████████████████████| 59/59 [03:18<00:00,  3.37s/it]
Used Signal Method: ICA
FFT MAE (FFT Label):4.766949152542373
FFT RMSE (FFT Label):16.699610764243513
FFT MAPE (FFT Label):4.4723324878843
FFT Pearson  (FFT Label):0.7210998217059267
===Signal Method ( POS ) Predicting ===
100%|███████████████████████████████████████████| 59/59 [02:26<00:00,  2.49s/it]
Used Signal Method: POS
FFT MAE (FFT Label):3.6720405190677967
FFT RMSE (FFT Label):11.822951673836915
FFT MAPE (FFT Label):7.248782385643253
FFT Pearson  (FFT Label):0.8783814244636281
===Signal Method ( CHROM ) Predicting ===
100%|███████████████████████████████████████████| 59/59 [01:33<00:00,  1.59s/it]
Used Signal Method: CHROM
FFT MAE (FFT Label):5.772477489406779
FFT RMSE (FFT Label):14.925515271353694
FFT MAPE (FFT Label):11.5169233509023
FFT Pearson  (FFT Label):0.809478372260371
===Signal Method ( GREEN ) Predicting ===
100%|███████████████████████████████████████████| 59/59 [01:33<00:00,  1.58s/it]
Used Signal Method: GREEN
FFT MAE (FFT Label):10.092525158898304
FFT RMSE (FFT Label):23.85435364661476
FFT MAPE (FFT Label):10.27803281768738
FFT Pearson  (FFT Label):0.34083764279755474
===Signal Method ( LGI ) Predicting ===
100%|███████████████████████████████████████████| 59/59 [01:36<00:00,  1.63s/it]
Used Signal Method: LGI
FFT MAE (FFT Label):4.610533633474576
FFT RMSE (FFT Label):15.383200109418059
FFT MAPE (FFT Label):4.963786008286058
FFT Pearson  (FFT Label):0.7716043755345129
===Signal Method ( PBV ) Predicting ===
100%|███████████████████████████████████████████| 59/59 [01:33<00:00,  1.58s/it]
Used Signal Method: PBV
FFT MAE (FFT Label):3.515625
FFT RMSE (FFT Label):11.861785559861659
FFT MAPE (FFT Label):4.474897898422112
FFT Pearson  (FFT Label):0.8632995963199497
(rppg-toolbox) [girishvn@g3005 rPPG-Toolbox]$ ```

### CODE REVIEW

**Comment by girishvn** [comment]
@xliucs @yahskapar

File: unsupervised_methods/unsupervised_predictor.py [hunk 1]
```diff
@@ -7,59 +7,59 @@
...
 
 
-def signal_predict(config, data_loader, method_name):
+def unsupervised_predict(config, data_loader, method_name):
     """ Model evaluation on the testing dataset."""
-    if data_loader["signal"] is None:
-        raise ValueError("No data for signal method predicting")
+    if data_loader["unsupervised"] is None:
+        raise ValueError("No data for unsupervised method predicting")
     print("===Signal Method ( " + method_name + " ) Predicting ===")
```
Comment (yahskapar) [inline unsupervised_methods/unsupervised_predictor.py:23]: Change this line to "Unsupervised Method" rather than "Signal Method"

File: config.py [hunk 2]
```diff
@@ -121,34 +121,34 @@
 # -----------------------------------------------------------------------------
 # Signal method settings
```
Comment (yahskapar) [inline config.py:122]: Change this comment to Unsupervised method settings

File: config.py [hunk 3]
```diff
@@ -121,34 +121,34 @@
...
 # -----------------------------------------------------------------------------\
-_C.SIGNAL = CN()
-_C.SIGNAL.METHOD = []
-_C.SIGNAL.METRICS = []
+_C.UNSUPERVISED = CN()
+_C.UNSUPERVISED.METHOD = []
+_C.UNSUPERVISED.METRICS = []
 # Signal.Data settings
```
Comment (yahskapar) [inline config.py:127]: Update this comment to Unsupervised.Data settings

File: config.py [hunk 4]
```diff
@@ -121,34 +121,34 @@
...
+_C.UNSUPERVISED.DATA.DATA_PATH = ''
+_C.UNSUPERVISED.DATA.EXP_DATA_NAME = ''
+_C.UNSUPERVISED.DATA.CACHED_PATH = 'PreprocessedData'
+_C.UNSUPERVISED.DATA.FILE_LIST_PATH = os.path.join(_C.UNSUPERVISED.DATA.CACHED_PATH, 'DataFileLists')
+_C.UNSUPERVISED.DATA.DATASET = ''
+_C.UNSUPERVISED.DATA.DO_PREPROCESS = False
+_C.UNSUPERVISED.DATA.DATA_FORMAT = 'NDCHW'
+_C.UNSUPERVISED.DATA.BEGIN = 0.0
+_C.UNSUPERVISED.DATA.END = 1.0
 # Signal Data preprocessing
```
Comment (yahskapar) [inline config.py:139]: Similar update to this comment

**Comment by yahskapar** [comment]
Few small fixes commented, LGTM otherwise tho

**Review by yahskapar — CHANGES_REQUESTED** [review]

File: config.py (hunk 4 above)
Comment (girishvn) [reply config.py]: Good catch. These have been resolved

**Review by yahskapar — APPROVED** [review]

File: config.py (hunk 4 above)
Comment (yahskapar) [reply config.py]: Great, thanks

**Pull request merged**